framework/src/media/audio/audio_manager.c

```diff
@@ -136,6 +136,7 @@ static audio_manager_result_t get_audio_volume(int fd, audio_config_t *config, a
 static audio_manager_result_t set_audio_volume(audio_card_type_t type, uint8_t volume);
 static audio_manager_result_t get_supported_sample_rate(int fd, uint32_t *sample_type, audio_card_type_t type);
 static uint32_t get_closest_samprate(uint32_t origin_samprate, audio_card_type_t type);
+static int pcm_writei_with_resampling(struct pcm *pcm, audio_card_info_t *cur_card, void *data, uint32_t frames);
```

Comment:
resample_streamout is better, I add comment continuosely.

Reply:
Modified.